src/main/java/frc/robot/subsystems/colour_wheel/ColourWheel.java

```diff
@@ -0,0 +1,94 @@
+package frc.robot.subsystems.colour_wheel;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.revrobotics.ColorMatch;
+import com.revrobotics.ColorMatchResult;
+import com.revrobotics.ColorSensorV3;
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+
+public class ColourWheel extends SubsystemBase {
+
+    private final TalonSRX motor = new TalonSRX(Ports.ColourWheel.MOTOR);
+
+    private final I2C.Port port = I2C.Port.kOnboard;
+
+    private final ColorSensorV3 colorSensorV3 = new ColorSensorV3(port);
+
+    private final ColorMatch colorMatch = new ColorMatch();
+
+    private String colorString = " ";

```

Comment:
```suggestion
    private String colorString = "";
```
You can use just an empty string.